probe-rs/targets/STM32U0_Series.yaml

```diff
@@ -0,0 +1,1651 @@
+  memory_map:
+  - !Ram
+    name: SRAM1
+    range:
+      start: 0x20000000
+      end: 0x20008000
+    cores:
+    - main
+  - !Nvm
+    name: Flash
+    range:
+      start: 0x8000000
+      end: 0x8040000
+    is_boot_memory: true
+    cores:
+    - main
+  - !Generic
+    name: SRAM2
+    range:
+      start: 0x20002000
+      end: 0x20004000
+    cores:
+    - main
+  flash_algorithms:
+  - stm32u0xx_256k
+flash_algorithms:
+- name: stm32u0xx_32k
+  description: STM32U0xx_32
+  default: true
+  instructions: ELUDRnK2jEiMTCBhIEYAagEkJAQgQAAoDdGJSIlMIGBAECBgBiBgYIdIoGDAEIdMYGB/ICBggUhAacAPwAcAKATQg0h9TKBggkigYAAgEL0BRnpIQGkBItIHEEN3SlBhACBwRwNGASBwR3K2AL9zSABpASEJBAhAACj40W9IQGkEIQhDbUlIYQhGQGkBIQkECENqSUhhAL9oSABpASEJBAhAACj40WVIAGljSQhAACgE0AhGYUkIYQEgcEdfSEBpBCGIQ11JSGEAIPbnAUZytllIWksYYRhGQGn/I9sAmENWS1hhASDABggaAgoYRkBpAiMTQxhDUUtYYRhGQGkBIxsEGENNS1hhAL9MSABpASMbBBhAACj40UhIAGlGSxhAACgE0BhGRUsYYQEgcEdDSEBpAiOYQ0FLWGEBIMAGAGhAHAAoBtEYRgBoASMbBBhDOksYYAAg6edwtYiwA0ZytsgdwQjJAAC/NEgAaQElLQQoQAAo+NE4SDBNKGEoRkBpASUoQy1NaGE/4AgpD9MQaBhgUGhYYAgzCDIIOQC/J0gAaQElLQQoQAAo+NEh4AAkBeAQeKUAbkZwUVIcZByMQvfTACQF4P8gZRitAG5GcFFkHAggQBqgQvXYAL8XSABpASUtBChAACj40QCYGGBYYAAhEkgAacAHwA8AKAXQDkgOTShhASAIsHC9ACm90QtIQGlACEAACU1oYQEgwAYAaEAcACgG0ChGAGgBJS0EqEMCTShgACDm5/rDAAAAIAJAqqoAAAAwAED/DwAAACwAQCMBZ0Wrie/N/wEAAAAAAAA=
+  load_address: 0x20000020
+  pc_init: 0x1
+  pc_uninit: 0x51
+  pc_program_page: 0x159
+  pc_erase_sector: 0xcd
+  pc_erase_all: 0x6b
+  data_section_offset: 0x25c
+  flash_properties:
+    address_range:
+      start: 0x8000000
+      end: 0x8008000
+    page_size: 0x400
+    erased_byte_value: 0xff
+    program_page_timeout: 400
+    erase_sector_timeout: 400
+    sectors:
+    - size: 0x800
+      address: 0x0
+- name: stm32u0xx_64k
+  description: STM32U0xx_64
+  default: true
+  instructions: gLWEsAOQApEBkgDwn/kVSRVICGAVSABowAMAKBLU/+cTSRRICGAUSAhgFEkGIAhgE0kUSAhgFEkUSAhgFEl/IAhg/+cTSABowA8AKAbQ/+cRSRJICGASSAhg/+cAIASwgL3ARhAgAkD6wwAAICACQAAwAECqqgAAVVUAAAQwAEAIMABA/w8AAAQsAED/AQAAACwAQBQgAkAIIAJAIwFnRauJ782BsACQBEkIaAEi0gcQQwhgACABsHBHwEYUIAJAg7ACkAGRaEYCcAEgA7BwR4C1grAA8Dr5/+cYSABowAMAKAHV/+f45xVJCGgEIhBDCGAIaAEiEgQQQwhg/+cPSABowAMAKAHV/+f45wtIAGgMSQhCBtD/5whJCkgIYAEgAZAH4AZJCGgEIpBDCGAAIAGQ/+cBmAKwgL3ARhAgAkAUIAJA+sMAAIC1hLACkADw/fgjSSNICGAjSQho/yLSAJBDCGACmB8i0gaAGAAKAZABmApoEEMCIhBDCGAIaAEiEgQQQwhg/+cVSABowAMAKAHV/+f45xJIAGgSSQhCBtD/5w9JD0gIYAEgA5AV4A5JCGgCIpBDCGABIMAGAGhAHAAoB9H/5wlJCGgBIhIEEEMIYP/nACADkP/nA5gEsIC9ECACQPrDAAAUIAJAACACQIC1jrAMkAuRCpIA8Kf4C5jAHQchiEMLkP/nS0gAaMADACgB1f/n+OdHSUhICGBISQhoASIQQwhg/+cLmAAoatD/5wuYCCga0//nCpgAaAyZCGAKmEBoDJlIYAyYCDAMkAqYCDAKkAuYCDgLkP/nNkgAaMADACgB1f/n+Oc+4AAgAZD/5wGYC5mIQg7S/+cKmAB4AZmKAAKpiFAKmEAcCpD/5wGYQBwBkOznACABkP/nAZgLmgghiRqIQgzS/+cBmAuZQBiCAAKp/yCIUP/nAZhAHAGQ7Of/5xtIAGjAAwAoAdX/5/jnApgMmQhggCECqEBYDJlIYAAgC5D/5xJIAGjABwAoBtD/5w9JE0gIYAEgDZAV4JHnDkoRaAEggUMRYMAGAGhAHAAoB9D/5wlJCGgBIhIEkEMIYP/nACANkP/nDZgOsIC9wEYQIAJA/wEAABQgAkAAIAJA+sMAAO/zEIDAB8APcrZwRwAAAAA=
```

Comment:
What's so special about 64K that they have a separate flash algo from the other ones... Also, the rest are identical, except for the flash range, maybe we should teach target-gen to merge them?